components/shared/ButtonGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import { ButtonHTMLAttributes, DetailedHTMLProps, type ReactElement } from "react";
+
+interface ButtonGroupProps {
+  label: string;
+  children: ReactElement<typeof ButtonGroupItem>[];
+}
+
+type ButtonGroupItemProps = DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> & {
+  variant?: "primary" | "secondary";
+};
+
+// TODO: get the proper bg colors.
```

Comment:
Just have to get the proper bg colors here @isabensusan. We can chat at our sync tomorrow.